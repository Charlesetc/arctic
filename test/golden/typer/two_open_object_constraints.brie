[
  define x x
  x.this
  x.that
]
