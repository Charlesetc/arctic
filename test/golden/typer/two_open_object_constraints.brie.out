{[4, 5]=>Unknown(), [6, 7]=>Unknown(), [0, 1, 2, 3, 8]=>Open_object(fields={"that"=>4, "this"=>6})}